packages/gatsby-plugin-inferno/src/gatsby-node.js

```diff
@@ -0,0 +1,15 @@
+exports.onCreateWebpackConfig = ({ stage, actions }) => {
+  // Requiring the server version of React-dom is hardcoded right now
+  // in the development server. So we'll just avoid loading Inferno there
```

Comment:
Was this copied from `preact` plugin or this is something new? I'm not sure what's the problem this check is workarounding

Reply:
`develop-html` is only for generating things for basic `index.html` used in develop mode. There is separate `develop` stage that actually produce js bundle, so not sure if this changes anything for hot-reload